File: Java Barcode Scanner ME/app/src/main/java/de/dimedis/mobileentry/fragments/VoidTicketFragment.java
package de.dimedis.mobileentry.fragments;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import de.dimedis.mobileentry.databinding.FragmentVoidticketBinding;

public class VoidTicketFragment extends BaseFragment {

    FragmentVoidticketBinding binding;

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        super.onCreateView(inflater, container, savedInstanceState);

        binding = FragmentVoidticketBinding.inflate(inflater, container, false);

        binding.ok.setOnClickListener(view -> back());
        binding.cancel.setOnClickListener(view -> back());
        return binding.getRoot();
    }

    public static Builder builder() {
        return new Builder();
    }

    public static class Builder {
        public Builder() {
        }

        public VoidTicketFragment build() {
            return new VoidTicketFragment();
        }
    }
}

File: Kotlin Migration/NTMobileEntry/app/src/main/java/de/dimedis/mobileentry/fragments/VoidTicketFragment.kt
package de.dimedis.mobileentry.fragments

import de.dimedis.mobileentry.databinding.FragmentVoidticketBinding

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

class VoidTicketFragment : BaseFragment() {
    var binding: FragmentVoidticketBinding? = null
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        super.onCreateView(inflater, container, savedInstanceState)
        binding = FragmentVoidticketBinding.inflate(inflater, container, false)
        binding!!.ok.setOnClickListener {
            back() }
        binding!!.cancel.setOnClickListener {
            back() }
        return binding!!.root
    }

    class Builder {
        fun build(): VoidTicketFragment {
            return VoidTicketFragment()
        }
    }

    companion object {
        fun builder(): Builder {
            return Builder()
        }
    }
}